customization/dispvm-customization.md

```diff
@@ -10,10 +10,64 @@ redirect_from:
 ---
 
 Disposable VM Customization
-====================
+============================
 
-Changing the DVM Template
--------------------------
+Qubes 4.0
+----------
+
+Disposable VM in Qubes 4.0 can be based on any template-based AppVM. You can also choose to use different AppVMs for different Disposable VMs. To prepare AppVM to be a base for Disposable VM, you need to set `template_for_dispvms` property, for example:
+
+    [user@dom0 ~]$ qvm-prefs fedora-26-dvm template_for_dispvms True
+
+Additionally, if you want to have menu entries for starting applications in Disposable VM based on this AppVM (instead of in the AppVM itself), you can achieve it with `appmenus-dispvm` feature:
+
+    [user@dom0 ~]$ qvm-features fedora-26-dvm appmenus-dispvm 1
+
+### Creating new Disposable VM base AppVM ###
+
+You may want to use multiple AppVMs for different Disposable VMs. To create new one, lets say `custom-dvm`, based on `debian-9` template, use following commands:
```

Comment:
> You may want to use multiple AppVMs for different Disposable VMs.

What does this mean, exactly?

Does it mean that I can start Disp1 based on AppVM1 and Disp2 based on AppVM2, rather than being forced to start both Disp1 and Disp2 based on AppVM1?

If that's what it means, I think it would be clearer to say:

"You may want to use different AppVMs for different Disposable VMs."

Reply:
It might be even better to say something like:

"In Qubes 4.0, you're no longer restricted to  a single DVM Template. Instead, you can create as many as you want. Whenever you start a new Disposable VM, you can choose to base it on whichever DVM Template you like."